ssl/internal.h

```diff
@@ -2804,6 +2813,12 @@ struct SSL3_STATE {
   // received.
   uint8_t warning_alert_count = 0;
 
+
+  // State of post handshake authentication. Can be any state from SSL_PHA_STATE
+  uint8_t pha_ext = 0;
```

Comment:
* Can we redefine this as `SSL_PHA_STATE ssl_pha_state = SSL_PHA_NONE`? The enum makes things clearer with what available states we have. Similar to what we do below with `ssl_ech_status_t ech_status = ssl_ech_none;`

* Minor spacing nit:
```suggestion
  // pha_ext indicates the state of post handshake authentication. It can
  // be any state from |SSL_PHA_STATE|
  uint8_t pha_ext = 0;
  
```
This follows the rest of the codebase documentation format.

Reply:
I agree -- use the enum you defined for this.

Reply:
That does make things clearer, made the change